css-view-transitions-1/Overview.bs

```diff
@@ -197,40 +197,55 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 # Pseudo-elements # {#pseudo}
 
-	The following <dfn export>view-transition pseudo-elements</dfn> represent the various items being animated.
+## Pseudo-element root ## {#pseudo-root}
 
-	The ''::view-transition'' pseudo-element acts as a grouping element for other [=view-transition pseudo-elements=]
-	and has the document's [=document element=] as its [=originating element=].
+	Issue: This definition should be moved to [[css-pseudo-4]].
 
-	Note: For example, '':root::view-transition'' selector matches this pseudo-element,
-	but ''div::view-transition'' does not.
+	A <dfn>pseudo-element root</dfn> is a type of [=tree-abiding pseudo-element=] that is the [=tree/root=] in a [=tree=] of [=tree-abiding pseudo-elements=],
+	known as the <dfn>pseudo-element tree</dfn>.
 
-	Other [=view-transition pseudo-elements=] take a <<pt-name-selector>> argument
-	to specify which elements named with 'view-transition-name' are affected.
+	The [=pseudo-element tree=] defines the document order of its [=tree/descendant=] [=tree-abiding pseudo-elements=].
 
-	There can be multiple pseudo-elements of the same type,
-	one for each 'view-transition-name' participating in a transition.
+	When a [=pseudo-element=] [=tree/participates=] in a [=pseudo-element tree=],
+	its [=originating pseudo-element=] is its [=tree/parent=].
 
-	The <<pt-name-selector>> is defined as follows:
+	If a [=tree/descendant=] |pseudo| of a [=pseudo-element root=] has no other [=tree/siblings=],
```

Comment:
For a moment I got confused about why we're limiting this to only descendants of root. Now I understand, child is used to refer to pseudo-elements which originate directly from the root and descendant is for any pseudo-element in root's tree.

Reply:
Yeah, I don't think `::view-transition:only-child` makes sense as a thing. Or, if we wanted it to be a thing, its definition would be more complicated.